config/packages/sulu_admin.yaml

```diff
@@ -29,6 +29,22 @@ sulu_admin:
                         icon: 'su-storage'
                         label: 'app.album_selection_label'
                         overlay_title: 'app.select_albums'
+            event_selection:
```

Comment:
@nnatter Would move this in an own `event_admin.yaml` so the whole config can be better copied from this PR.

Reply:
would you do the same for the `Album` entity then? 🤔

Reply:
Is both for me okay, to let it in sulu_admin or have an own config for it, just want to make it easier to copy the config from the PR here.